docs/Getting Started.md

```diff
@@ -0,0 +1,43 @@
+# Clone Serai
+
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+# Build and Run Serai
+
+- Ubuntu LTS or WSL2 w/ Docker (Other distributions may also work or docker can be used)
+    - Windows Guide: [https://dev.to/davidkou/running-docker-in-windows-subsystem-for-linux-wsl2-1k43](https://dev.to/davidkou/running-docker-in-windows-subsystem-for-linux-wsl2-1k43)
+    - [https://docs.docker.com/engine/install/ubuntu/](https://docs.docker.com/engine/install/ubuntu/)
+- Setup Rust & Dependencies (Assuming Linux or WSL2)
+  - Install Rust  `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+  
+## Install Solidity Compiler
+```
+     sudo pip3 install solc-select
+     solc-select install 0.8.16
+     solc-select use 0.8.16
+```
+## Install Other Dependencies
+```sudo apt-get install -y \
+    cmake \
+    libboost-all-dev \
+    pkg-config \
+    libssl-dev
+```

```

Comment:
I don't believe these are required. We no longer build Monero so they shouldn't be. We may still need clang though... (but I believe that's shipped with Ubuntu)

Reply:
This is also not on a newline within the code block, yet then the last ones here don't have any ident at all :thinking: I'd be fine with dropping all identing and just using code on its own lines.

Reply:
Pretty sure build errors led to me adding these dependencies versus me keeping them for no reason. Can check again on a fresh install? Otherwise let's keep them and get feedback from devs as they go through the process.

Reply:
It could be the GitHub CI provides them, but I do recognize these as Monero depends. The Dockerfile PR had the report that it needed clang, not previously specified, due to GH CI provision yet you also didn't include that lol. So not sure.